apps/nextjs-website/src/components/atoms/UserInfo/UserInfo.tsx

```diff
@@ -18,14 +57,17 @@ const UserInfo: FC = () => {
       flexGrow={1}
       alignItems='center'
       direction='row'
-      gap={2}
+      gap={1}
       justifyContent='flex-end'
     >
       <Hidden smDown>
-        <Typography variant='body2'>Hello {user?.attributes?.email}</Typography>
+        <PersonOutline />
+        <Typography variant='body2' sx={{ fontWeight: 600, fontSize: '14px' }}>
+          {user.attributes.given_name} {user.attributes.family_name}
+        </Typography>
       </Hidden>
       <Button size='small' variant='contained' onClick={signOut}>
-        Sign out
+        {logout}
       </Button>
```

Comment:
I think this button should be a little smaller becouse now change hedere height. This can ba an issue for sticky header in other pages
[Screencast from 15-11-2023 15:49:22.webm](https://github.com/pagopa/developer-portal/assets/24475319/ad4a0108-e88e-4298-9a69-68e0b6b8c918)